roles/nuage_master/handlers/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
-- name: restart nuage-openshift-monitor
-  become: yes
-  systemd: name=nuage-openshift-monitor state=restarted
+- name: restart master
```

Comment:
This handler and references to it can be removed. The single master service was removed in https://github.com/openshift/openshift-ansible/pull/4832.